src/cascadia/TerminalApp/CommandSerialization.cpp

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "Command.h"
+#include "Utils.h"
+#include "ActionAndArgs.h"
+#include <LibraryResources.h>
+
+using namespace winrt::Microsoft::Terminal::Settings;
+using namespace winrt::TerminalApp;
+
+static constexpr std::string_view NameKey{ "name" };
+static constexpr std::string_view IconPathKey{ "iconPath" };

```

Comment:
```suggestion
static constexpr std::string_view IconPathKey{ "icon" };
```
The path is implied?